src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty/config.jelly

```diff
@@ -0,0 +1,22 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<f:section title="Kubernetes Plugin">
+<j:if test="${instance != null}">
+	<f:advanced>
+	<f:entry title="Usage restricted Kubernetes Clouds available to any Pipeline jobs inside this folder:"/>
+	<j:forEach var="permission" items="${instance.getEffectivePermissions()}">
+		<f:entry title="${permission.name}"><j:expr value="${permission.htmlCheckbox()}" escapeText="false" /></f:entry>
```

Comment:
Why not use <f:checkbox> here?

Reply:
couldn't figure out how to get the checkbox visually rendered as disabled in jelly.
by using readonly it looked like an active checkbox. i thought this would be confusing for the user.